docs/userguide/campaigns.md

```diff
@@ -1,25 +1,125 @@
 # Campaigns
 
-## General information
+Campaigns play a crucial role in Design of Experiments, and consequently also for BayBE.
+They serve as a structured framework for defining and documenting an experimentation
+process. The [`Campaign`](baybe.campaign.Campaign) class is used to model camaings and 
+serves as the primary interface for interacting with BayBE.
+While creating a campaign can be as simple as specifying a search space with experimental
+parameters and an optimization objective, there are numerous additional aspects that can
+be customized and fine-tuned.
 
-[`Campaigns`](baybe.campaign.Campaign) are the central objects of BayBE. They are used to organize the different aspects and are the main point of interaction via python.
+```{note}
```

Comment:
hmm in the interest of saving space imo this note is quite unneccessary, we dont need to announce that we will provide links

Reply:
Removed.